common/src/main/java/com/cinemamod/mcef/ModScheme.java

```diff
@@ -0,0 +1,116 @@
+package com.cinemamod.mcef;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import com.mojang.logging.LogUtils;
+import org.cef.callback.CefCallback;
+import org.cef.handler.CefResourceHandler;
+import org.cef.misc.IntRef;
+import org.cef.misc.StringRef;
+import org.cef.network.CefRequest;
+import org.cef.network.CefResponse;
+
+// https://github.com/CinemaMod/mcef/blob/master-1.19.2/src/main/java/net/montoyo/mcef/example/ModScheme.java
+public class ModScheme implements CefResourceHandler {
+    private String contentType = null;
+    private InputStream is = null;
+
+    String url;
```

Comment:
Make private?

Reply:
I think protected probably makes more sense, incase a mod wants to extend that scheme
But then again, if I were to do that then the other things should also be protected and those don’t make as much sense to be protected, so I guess yeah